dir/path.go

```diff
@@ -77,7 +76,24 @@ const (
 	TrustStoreDir = "truststore"
 )
 
-var userConfigDir = os.UserConfigDir // for unit test
+// The relative path to {NOTATION_LIBEXEC}
+const (
+	// PathPlugins is the plugins directory relative path.
+	PathPlugins = "plugins"
```

Comment:
Why we need to export it?

Reply:
It's not exported by this PR. It was in the wrong place.